modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/application/DefaultApplicationFactory.java

```diff
@@ -125,6 +126,12 @@ public ApplicationDescriptor createArtifactDescriptor(File artifactLocation, Opt
   }
 
   public Application createArtifact(ApplicationDescriptor descriptor) throws IOException {
```

Comment:
should this be deprecated? are there any remaining usages of this method?

Reply:
the unique consumer is tooling services and should use the new one. So this method can be deprecated